docs/docs/behaviors/hold-tap.md

```diff
@@ -48,6 +48,28 @@ If you press a tapped hold-tap again within `quick-tap-ms` milliseconds, it will
 
 In QMK, unlike ZMK, this functionality is enabled by default, and you turn it off using `TAPPING_FORCE_HOLD`.
 
+#### `global-quick-tap`
+
+If global quick tap is enabled, then the quick tap will apply not only when the given hold tap is tapped, but for any key tap before it. This effectively disables the hold tap when typing quickly, which can be quite useful for home row mods. It can also have the effect of removing the input delay when typing quickly.
+
+For example, the following hold tap configuration enables global quick tap with a 125 millisecond term.
+```
+gqt: global-quick-tap {
+	compatible = "zmk,behavior-hold-tap";
+	label = "GLOBAL_QUICK_TAP";
+	#binding-cells = <2>;
+	flavor = "tap-preferred";
+	tapping-term-ms = <200>;
+	quick-tap-ms = <125>;
+	global-quick-tap;
+	bindings = <&kp>, <&kp>;
+};
+```
+
+If you press `&kp A`, and then `&gqt LEFT_SHIFT B` **within** 125 ms after, then `ab` will be output. Importantly, the `b` will be output immediately since it was within the `quick-tap-ms`. This behavior will hold for any key that is output to the computer. The `&gqt LEFT_SHIFT B` binding will only have its underlying hold-tap behavior if it is pressed 125 ms **after** any keycode is sent to the computer.
+
+Note that the higher the `quick-tap-ms`, the harder it will be to use the hold behavior, making this unideal for capitalizing letter while typing normally. However, paired with [mod-tap](mod-tap.md)
```

Comment:
This seems to be missing a part.

Reply:
Yikes, I forgot to save the file, I actually had a different blurb that was supposed to be there.